requirements/js_test.txt

```diff
@@ -4,19 +4,18 @@
 #
 #    pip-compile --output-file requirements/js_test.txt requirements/js_test.in
 #
-argparse==1.4.0           # via jasmine
 cheroot==6.0.0            # via cherrypy
-cherrypy==13.1.0          # via jasmine
+cherrypy==14.0.0          # via jasmine
 glob2==0.6                # via jasmine-core
-jasmine-core==2.9.1       # via jasmine
-jasmine==2.9.0
+jasmine-core==3.0.0       # via jasmine
```

Comment:
you might want to consider pinning the jasmine version in js_text.in - looks like this is probably why the jasmine test suite isn't running.

Reply:
Yeah, reverting back to the original versions fixed the issue in the CI. I will add the jasmine version into js_text.in to pin it, though.